layouts/shortcodes/flexcolumns.html

```diff
@@ -0,0 +1,18 @@
+<div class="book-columns flex flex-wrap">
+{{ if .Get "ratio" }}
+  {{ $ratioSplit := split (.Get "ratio") ":"}}
+  {{ range $i, $val := split .Inner "<--->" }}
+    {{ if gt (len $ratioSplit) $i}}
+      <div style="flex-grow:{{ index $ratioSplit $i }};" class="flex-even markdown-inner">
+        {{ $val | $.Page.RenderString }}
+      </div>
+    {{ end }}
+  {{ end }}
+{{ else }}
+  {{ range split .Inner "<--->" }}
+  <div class="flex-even markdown-inner">
+    {{ . | $.Page.RenderString }}
+  </div>
+  {{ end }}
+{{ end }}
+</div>
```

Comment:
nit: please add a trailing newline

Reply:
Done (and I added it to the other shortcodes).